test/cmake/seqan3_require_benchmark.cmake

```diff
@@ -0,0 +1,50 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+# Exposes the google-benchmark target `gbenchmark`.
+macro (seqan3_require_benchmark)
+    enable_testing ()
```

Comment:
I just saw it in other places as well. Somehow this looks strange if the parenthesis is separated with a blank if you compare it to c++ code. Is this really the recommended style for cmake?

Reply:
Ask Hannes, he was in favour for this. cmake advertises this spacing in https://cmake.org/examples/, but uses a different one in the documentation: https://cmake.org/cmake/help/v3.18/manual/cmake-compile-features.7.html#compile-feature-requirements.

I was more in favour for `macro(seqan3_require_benchmark)`, `enable_testing()`, `if(hello)`, `endif()` and so on

Reply:
The point is, it is not consistent and given the style in C++ it is always awkward to see it differently here. But not so important.